### DISCUSSION TITLE
Auto apply filters from the filter panel

### DISCUSSION THREAD

**Discussion by maximStafeevAtAllscripts** [apache/superset#29628] [Q&A / Help]
Every time I and users forget to press "Apply filters" button in the filters panel. Is there any ability to apply filters automatically just after entering any value?

**Comment by sfirke** [comment]
This is not currently possible unfortunately.  There is an existing GitHub Discussion here that I believe covers this case: #20663.  I am going to close this Q&A since there's a feature request elsewhere already, feel free to follow along with that post or add additional details if you have them.
